### PR TITLE
[Elm] [1.0] reprocess access package assignment

### PR DESCRIPTION
**Pull request by ajburnle**
doc update because this api is now in 1.0

### CODE REVIEW

**Comment by ajburnle** [comment]
@FaithOmbongi  @Lauragra it won't let me change the merge labels. Can you please review this pull request

**Comment by ajburnle** [comment]
assign: @ajburnle

**Comment by ajburnle** [comment]
reassign: @FaithOmbongi

**Comment by ajburnle** [comment]
re-assign: @FaithOmbongi

File: api-reference/beta/api/accesspackageassignment-reprocess.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The following is an example of the request.
   "name": "reprocess_accesspackageassignments"
 }-->
 ```http
-POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackageAssignments/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
+POST https://graph.microsoft.com/v1.0/identityGovernance/entitlementManagement/accessPackageAssignments/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignment-reprocess.md:63]: This is the beta doc - so beta endpoint
```suggestion
POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackageAssignments/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```

File: api-reference/beta/api/accesspackageassignmentrequest-reprocess.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 In [Azure AD entitlement management](../resources/entitlementmanagement-overview.md), callers can automatically retry a user's request for access to an access package. It is performed on an [accessPackageAssignmentRequest](../resources/accesspackageassignmentrequest.md) object whose **requestState** is in a `DeliveryFailed` or `PartiallyDelivered` state. 
 
+You can only reprocess a request up to 14 days from the time the original request was completed. For requests completed more than 14 days, you can't reprocess those requests. You will need to ask the users to cancel the request(s) and make a new request in MyAccess.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignmentrequest-reprocess.md:18]: Suggestion
```suggestion
You can only reprocess a request within 14 days from the time the original request was completed. For requests completed more than 14 days, you will need to ask the users to cancel the request(s) and make a new request in the [MyAccess](http://myaccess.microsoft.com/) portal.
```

File: api-reference/beta/api/accesspackageassignmentrequest-reprocess.md [hunk 3]
```diff
@@ -60,7 +62,7 @@ The following is an example of the request.
   "name": "reprocess_accesspackageassignmentrequest"
 }-->
 ```http
-POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackageAssignmentRequests/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
+POST https://graph.microsoft.com/v1.0/identityGovernance/entitlementManagement/accessPackageAssignmentRequests/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignmentrequest-reprocess.md:65]:
```suggestion
POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackageAssignmentRequests/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```

File: api-reference/v1.0/resources/accesspackageassignment.md [hunk 4]
```diff
@@ -18,18 +18,24 @@ In [Azure AD Entitlement Management](entitlementmanagement-overview.md), an acce
...
+|accessPackageId|String|The identifier of the access package. Read-only.|
+|assignmentPolicyId|String|The identifier of the access package assignment policy. Read-only.|
+|assignmentState|String|The state of the access package assignment. Possible values are `Delivering`, `Delivered`, or `Expired`. Read-only. Supports `$filter` (`eq`).|
+|assignmentStatus|String|More information about the assignment lifecycle.  Possible values include `Delivering`, `Delivered`, `NearExpiry1DayNotificationTriggered`, or `ExpiredNotificationTriggered`.  Read-only.|
+|catalogId|String|The identifier of the catalog containing the access package. Read-only.|
+|expiredDateTime|DateTimeOffset|The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
+|id|String| Read-only.|
+|isExtended|Boolean|Indicates whether the access package assignment is extended. Read-only.|
+|targetId|String| The ID of the subject with the assignment. Read-only.|
+|schedule|[requestSchedule](requestschedule.md)| When the access assignment is to be in place. Read-only.|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/accesspackageassignment.md:38]: Hmm - The schema for v1.0 looks different from the schema for beta in that only 4 properties are surfaced in v1.0. Could you confirm why the difference?

File: api-reference/v1.0/resources/accesspackageassignmentrequest.md [hunk 5]
```diff
@@ -22,13 +22,15 @@ In [Azure AD Entitlement Management](entitlementmanagement-overview.md), an acce
...
 |[cancel](../api/accesspackageassignmentrequest-cancel.md)|[accessPackageAssignmentRequest](../resources/accesspackageassignmentrequest.md) collection|Cancel an **accessPackageAssignmentRequest** object that is in a cancellable state.|
+| [reprocess](../api/accesspackageassignmentrequest-reprocess.md) | None | Automatically retry a user’s request for access to an access package.|
 
 ## Properties
 |Property|Type|Description|
 |:---|:---|:---|
 |completedDateTime|DateTimeOffset|The date of the end of processing, either successful or failure, of a request. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. Read-only.|
 |createdDateTime|DateTimeOffset|The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. Read-only.|
 |id|String|Read-only.|
+|requestState|String|One of `PendingApproval`, `Canceled`, `Denied`, `Delivering`, `Delivered`, `PartiallyDelivered`, `DeliveryFailed`, `Submitted` or `Scheduled`. Read-only.|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/accesspackageassignmentrequest.md:33]: Property not in the v1.0 schema. Kindly confirm

File: api-reference/v1.0/api/accesspackageassignment-reprocess.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+### Request
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "ignored",
+  "name": "reprocess_accesspackageassignments"
+}-->
+```http
+POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackageAssignments/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/accesspackageassignment-reprocess.md:61]: This is the v1.0 doc
```suggestion
POST https://graph.microsoft.com/v1.0/identityGovernance/entitlementManagement/accessPackageAssignments/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```

File: api-reference/v1.0/api/accesspackageassignmentrequest-reprocess.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+### Request
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "ignored",
+  "name": "reprocess_accesspackageassignmentrequest"
+}-->
+```http
+POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackageAssignmentRequests/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/accesspackageassignmentrequest-reprocess.md:61]:
```suggestion
POST https://graph.microsoft.com/v1.0/identityGovernance/entitlementManagement/accessPackageAssignmentRequests/d82eb508-acc4-43cc-bcf1-7c1c4a2c073b/reprocess
```

File: changelog/Microsoft.DirectoryServices.json [hunk 8]
```diff
@@ -1,5 +1,31 @@
...
+          "Description": "Added the [reprocess](https://docs.microsoft.com/en-us/graph/api/reprocess?view=graph-rest-1.0) method to the [reprocess](https://docs.microsoft.com/en-us/graph/api/reprocess?view=graph-rest-1.0) resource.",
+          "Target": "reprocess"
+        },
+        {
+          "Id": "6c83378e-32f7-4a90-be50-fba95f7890ae",
+          "ApiChange": "Method",
+          "ChangedApiName": "reprocess",
+          "ChangeType": "Addition",
+          "Description": "Added the [reprocess](https://docs.microsoft.com/en-us/graph/api/reprocess?view=graph-rest-1.0) method to the [reprocess](https://docs.microsoft.com/en-us/graph/api/reprocess?view=graph-rest-1.0) resource.",
+          "Target": "reprocess"
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:19]: Fixed links
```suggestion
          "Description": "Added the [reprocess](https://docs.microsoft.com/en-us/graph/api/reprocess?view=graph-rest-1.0) method to the [accessPackageAssignment](https://docs.microsoft.com/en-us/graph/api/accessPackageAssignment?view=graph-rest-1.0) resource.",
          "Target": "accessPackageAssignment"
        },
        {
          "Id": "6c83378e-32f7-4a90-be50-fba95f7890ae",
          "ApiChange": "Method",
          "ChangedApiName": "reprocess",
          "ChangeType": "Addition",
          "Description": "Added the [reprocess](https://docs.microsoft.com/en-us/graph/api/reprocess?view=graph-rest-1.0) method to the [accessPackageAssignmentRequest](https://docs.microsoft.com/en-us/graph/api/accessPackageAssignmentRequest?view=graph-rest-1.0) resource.",
          "Target": "accessPackageAssignmentRequest"
```

File: changelog/Microsoft.DirectoryServices.json [hunk 9]
```diff
@@ -1,5 +1,31 @@
...
+        }
+      ],
+      "Id": "6c83378e-32f7-4a90-be50-fba95f7890ae",
+      "Cloud": "Prod",
+      "Version": "v1.0",
+      "CreatedDateTime": "2022-03-25T02:09:36.1935694Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Identity governance"
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:27]:
```suggestion
      "SubArea": "Governance"
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @ajburnle for your work in this PR. I've left suggestions for minor fixes.

One major discrepancy is the schema vs this PR - the released v1.0 schema doesn't contain any of the properties you've added to the two resources. Could you confirm why the difference between beta and v1.0?

File: api-reference/beta/api/accesspackageassignment-reprocess.md (hunk 1 above)
Comment (ajburnle) [reply api-reference/beta/api/accesspackageassignment-reprocess.md:63]: no we want it to be a 1.0 doc

**Comment by FaithOmbongi** [comment]
Hi @ajburnle - This PR has been overtaken by https://github.com/microsoftgraph/microsoft-graph-docs/pull/17585 I believe. Please close it.

**Comment by FaithOmbongi** [comment]
> This PR has been overtaken by #17585 I believe. Please close it.

Closing this PR.

**Pull request closed without merging**